hax/hax/filestats.py

```diff
@@ -47,7 +47,6 @@ def stop(self) -> None:
     def _execute(self, motr: Motr):
         try:
             LOG.info('filesystem stats updater thread has started')
-            motr.adopt_motr_thread()
```

Comment:
`hare.spec` file changes should have no impact as the changes are in if/else for OS ver.
But for these changes which are OS independent, will need some testing on our current platform - CentOs 7.9.
Looks like testing was done for CentOs 8.
Just wanted to confirm if testing was done for 7.9 as well.

Thanks!

Reply:
I suppose the testing on 7.9 is done by our CI?

The `hax` was crashing on the very startup, so if it does not crash in startup in 7.9 during the CI - that's enough testing, I suppose.

Reply:
ok.
Yes automated VM deployment on 7.9 here is passed.